### DISCUSSION TITLE
Do not hide thread for one answer or accepted answer

### DISCUSSION THREAD

**Discussion by Gervwyk** [community/community#3579] [Discussions]
Hi!

Loving Github discussion so far for [Lowdefy](https://github.com/lowdefy/lowdefy/discussions/501)

One question, does it make sense to hide a thread when there is only one answer in the thread? [Consider](https://github.com/lowdefy/lowdefy/discussions/501)

Or also then, does it make sense to hide the accepted answer? 

It makes sense for lots of answers on a page, but most pages will only have one I think.

**Comment by robertogl** [comment]
I had the same comment! I think a single Q&A discussion should have the option to host multiple question threads and therefore multiple answers.

**Comment by PranamBhat** [comment]
Hi @Gervwyk 

For **Question** and **Answer** scenario it is a good practice.

However, GitHub is having "**Discussions**" which is meant to be a discussion rather than only Q&A. So your scenario won't come to picture here.

Thank you.
